tests/unit/ground-commander-test.cpp

```diff
@@ -34,24 +34,29 @@
 #define CMD_BUF_SIZE 25 // TODO this should be globally defined
 
 static char command_buf[CMD_BUF_SIZE] = {'\0'};
-#define SPACE_COMMANDER_BIN  "bin/space-commander/space-commander" // use local bin, not the one unser CS1_APPS
+
 #define GROUND_COMMANDER_BIN "bin/ground-commander/ground-commander" // use local bin, not the one unser CS1_APPS
 
 #define UTEST_SIZE_OF_TEST_FILES 6
 
-TEST_GROUP(GroundCommanderTestGroup)
+TEST_GROUP(GroundCommandTestGroup)
```

Comment:
I think this should read: GroundCommanderTestGroup 

Because it is for the ground-commander